config/xiaomi-ysl.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+
+OPTIONS=(-r vsn -r vsp)
+PANELS=(
+	[auo_hd720_video]="mdss,auo-hd720"
+	[sc_ili9881c_hd720_video]="mdss,sc-ili9881c-hd720"
+	[truly_hd720_video]="mdss,truly-hd720"
+)
```

Comment:
Do you think all of these panels actually exist on phone and are not just leftovers from Qualcomm base code?

Reply:
https://github.com/MiCode/Xiaomi_Kernel_OpenSource/commit/0884128ae3ddf66f604d9f2beb60a442cae9aba5#diff-2f06de2c1b08a57c699585eda304c60418c6f6024a3b1d2ae63c7de942d3e986

Reply:
looks good then.

But please change compatibles to the following:

* `mdss,auo-hd720` to `xiaomi,ysl-ili7807d`
* `mdss,sc-ili9881c-hd720` to `xiaomi,ysl-ili9881c`
* `mdss,truly-hd720` to `xiaomi,ysl-hx8394f`

Names are from the downstream dtsi, e.g. https://github.com/MiCode/Xiaomi_Kernel_OpenSource/blob/0884128ae3ddf66f604d9f2beb60a442cae9aba5/arch/arm/boot/dts/qcom/dsi-panel-truly-hd720-video.dtsi#L16